src/main/java/seedu/address/logic/commands/results/CommandResult.java

```diff
@@ -8,52 +8,33 @@
  * Represents the result of a command execution.
  */
 public class CommandResult {
-
-    // TODO: OOP to make more CommandResult classes
-
-    private final String feedbackToUser;
-    private final String dataToUser;
-    private final String displayType;
-
     /** Preview information about a resume. */
-    private final boolean isShowPreview;
+    protected boolean isShowPreview;
```

Comment:
Should we set these values with initial value of false? I'm scared of a null returning when it's supposed to be a boolean. But actually iirc, if (null) is false.. so it's probably okay.

Reply:
Good idea. Working on it!